pkgs/build-support/setup-hooks/wrap-gapps-hook/wrap-gapps-hook.sh

```diff
@@ -11,7 +11,7 @@ addEnvHooks "${targetOffset:?}" find_gio_modules
 
 gappsWrapperArgsHook() {
     if [ -n "$GDK_PIXBUF_MODULE_FILE" ]; then
-        gappsWrapperArgs+=(--set GDK_PIXBUF_MODULE_FILE "$GDK_PIXBUF_MODULE_FILE")
+        gappsWrapperArgs+=(--suffix GDK_PIXBUF_MODULE_FILE : "$GDK_PIXBUF_MODULE_FILE")
```

Comment:
I wonder if `--prefix` might be safer here. E.g. in case user has `GDK_PIXBUF_MODULE_FILE` in their environment containing ABI incompatible loaders (assuming the cache format is stable, first registered module for given file takes precedence, and modules are loaded lazily). But it is not clear to me what handles matching the image to a module, so :woman_shrugging: